Authentication/SessionServiceType.swift

```diff
@@ -94,7 +94,7 @@ public protocol SessionServiceType {
          Avoid sending a .LogIn event as well.
          Don't forget to update the currentUser.
     */
-    func signUp(name: String, email: Email, password: String) -> SignalProducer<User, SessionServiceError>
+    func signUp(name: String?, email: Email, password: String) -> SignalProducer<User, SessionServiceError>
```

Comment:
None the less, we should decide to use username or name and I will change everything to one of them.
@guidomb